packages/utilities/src/FocusRectsProvider.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { FocusRectsContext } from './useFocusRects';
+
+export type FocusRectsProviderParams = {
+  /**
+   * Ref to the root element that this is providing focus rects for.
+   */
+  providerRef: React.RefObject<HTMLElement>;
+
+  /**
+   * Indicates that this is the root of a layer, and should not inherit the providerRef from a parent context.
+   */
+  layerRoot?: boolean;
+};
+
+export const FocusRectsProvider: React.FC<FocusRectsProviderParams> = params => {
```

Comment:
nit: I expect this argument to be called `props`. Is there a reason it's called `params`?

Reply:
Heh good call. I'd originally made this a hook and called the argument params... but later changed it to be a component and forgot to rename it.